stories/FullScreen.stories.js

```diff
@@ -0,0 +1,29 @@
+import React, { useRef, useEffect, useState } from 'react';
+import FullScreenButton from '../src/components/SimpleCommonComponents/FullScreenButton';
+
+import { foundationPartial } from '../src/styles/foundation';
+import fonts from 'EBI-Icon-fonts/fonts.css';
+const f = foundationPartial(fonts);
+export default {
+  title: 'Basic UI/FullScreen',
+};
+
+export const Basic = () => {
```

Comment:
The FullScreenButton has a two attributes to _hook_ functionality before entering to fullscreen mode and after exiting... I think it would be useful to have a second story that uses them for something basic... maybe changing the background color.